src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/tests/Toolbar.test.js

```diff
@@ -4,20 +4,74 @@ import {render} from 'enzyme';
 import Toolbar from '../Toolbar';
 import toolbarStore from '../stores/ToolbarStore';
 
-jest.mock('../stores/ToolbarStore', () => ({}));
+jest.mock('../stores/ToolbarStore', () => ({
+    hasBackButtonConfig() {
+        return false;
+    },
+
+    getBackButtonConfig() {
+        return this.config.backButton || null;
+    },
+
+    hasItemsConfig() {
+        return false;
+    },
+
+    getItemsConfig() {
+        return this.config.items || [];
+    },
+
+    hasIconsConfig() {
+        return false;
+    },
+
+    getIconsConfig() {
+        return this.config.icons || [];
+    },
+
+    hasLocaleConfig() {
+        return false;
+    },
+
+    getLocaleConfig() {
+        return this.config.locale;
+    },
+}));
 
 test('Render the items from the ToolbarStore', () => {
-    toolbarStore.items = [
-        {
-            title: 'Save',
-            enabled: false,
-            icon: 'save',
-        },
-        {
-            title: 'Delete',
-            icon: 'delete',
-        },
-    ];
+    toolbarStore.hasItemsConfig = jest.fn();
```

Comment:
I'd probably do the same for all other methods. This way you force the implementor of the test to build up his own configuration, and not rely on the already existing one. This causes a lot of tests to fail when you want to update the mock.